src/legacy/core_plugins/visualizations/public/embeddable/visualize_embeddable_factory.tsx

```diff
@@ -192,18 +168,16 @@ export class VisualizeEmbeddableFactory extends EmbeddableFactory<
   public async create() {
     // TODO: This is a bit of a hack to preserve the original functionality. Ideally we will clean this up
     // to allow for in place creation of visualizations without having to navigate away to a new URL.
-    if (this.visTypes) {
-      showNewVisModal(
-        this.visTypes,
-        {
-          editorParams: ['addToDashboard'],
-        },
-        npStart.core.http.basePath.prepend,
-        npStart.core.uiSettings,
-        npStart.core.savedObjects,
-        npSetup.plugins.usageCollection
-      );
-    }
+    showNewVisModal(
```

Comment:
Are we covered for the case where `visTypes` used to be `undefined`?
Can we possibly reach this code in Angular without having types initialized?

Reply:
getTypes will already procude an error if visTypes is undefined, but this should never happen (if it does there is an error in code and something is trying to access visTypes before plugin system has stated)